src/main/java/athena/task/Time.java

```diff
@@ -104,6 +104,7 @@ public void setRecurrence(String recurrence) {
             } catch (DateTimeParseException e) {
                 // TODO: Handle this properly
                 System.out.println("I don't understand the date you gave. So I set it to today.");
+                this.recurrence = LocalDate.now().getDayOfWeek().toString();
```

Comment:
If we do `getDayOfWeek` means it will occur every week right?

Reply:
okayy thanks! will make the change